src/scicookie/hooks/post_gen_project.py

```diff
@@ -111,19 +126,15 @@ def remove_package_file(filepath: str):
 
 def move_selected_doc_dir():
     if DOCUMENTATION_ENGINE == "mkdocs":
-        docs_target_dir = PROJECT_DIRECTORY
```

Comment:
@xmnlab Since I deleted the references files and changed the folders I thought I should remove all of that from that function, but I saw that I didn't. Let me put it back